est/001.est

```diff
@@ -0,0 +1,20 @@
+date: 28-02-2016
+author: Dali Freire
+method: champions.pert
+scope:
+  1: Infrastructure setup
```

Comment:
@dalifreire I think given that we cleared up point 2 now, we need to break this up into pieces.
Otherwise you give 3 tasks for creating production code:
* Infrastructure Setup
* Operation Commands
* Project Site

The first one is a lot more general than the other 2 and the est readme clearly states

> It is important to keep all work items on the same level of abstraction

I think if we keep that in mind in 1. and break it apart, we're good, the rest looks ok now :)